model_analyzer/triton/server/server_docker.py

```diff
@@ -62,11 +62,9 @@ def start(self):
         Starts the tritonserver docker container using docker-py
         """
 
-        if len(self._gpus) == 1 and self._gpus[0] == 'all':
-            devices = [
-                docker.types.DeviceRequest(count=-1, capabilities=[['gpu']])
-            ]
-        else:
+        # List GPUs to be mounted inside docker container
+        devices = []
+        if len(self._gpus):
```

Comment:
Is this case now taken care of after calling verify_requested_gpus?

Reply:
Yes. The `verify_requested_gpus` will be called by the `server_factory.py` before calling into `server_docker.py` and `server_local.py`.